.github/workflows/wheel_noarch.yml

```diff
@@ -93,12 +93,6 @@ jobs:
           name: pure-python-wheels-${{ matrix.pl_backend }}.zip
           path: dist/*.whl
 
-      - uses: actions/upload-artifact@v3
-        if: ${{ matrix.pl_backend != 'lightning_qubit' && (github.event_name == 'release' || github.ref == 'refs/heads/master') }}
-        with:
-          name: pure-source-dist-${{ matrix.pl_backend }}.tar.gz
-          path: dist/*.tar.gz
-
```

Comment:
What's the reason behind this? We still care about uploading sdists :)

Reply:
Yes, but it seems we're skipping the build steps altogether. Let's fix in a follow-up PR.